schema/composition/animation.json

```diff
@@ -10,6 +10,12 @@
         {
             "type": "object",
             "properties": {
+                "sv": {
```

Comment:
I'd prefer something more human-readable/suggestive, but not sure how to best avoid ambiguity vs. the existing `v` prop.

Some ideas: `ver`, `version`, or plain `lottie`?

Reply:
Yeah, I'm not sure the best way to disambiguate the two either. Maybe documenting the existing field and marking it deprecated would help? I'm guessing it is too risky to re-use that field, but could be worth exploring.

Reply:
Slept on this, and like `ver`. Still compact and is a good long term name, regardless of how we resolve existing conflict.

Reply:
Thanks, lgtm!